.github/workflows/node.js.yml

```diff
@@ -24,5 +24,5 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm run build
+    - run: yarn install --frozen-lockfile
+    - run: yarn build
```

Comment:
Are test commands missing?

Reply:
Added.

Reply:
We cannot add a test command now. I will make a new PR for it. Some tests are failing we will need to fix those first.